python/ray/dag/tests/experimental/test_accelerated_dag.py

```diff
@@ -2297,6 +2322,25 @@ def call(self, value):
     assert ray.get(ref) == 3
 
 
+def test_driver_as_actor_and_actor_reading(ray_start_cluster):
+    @ray.remote
+    class Replica:
+        def __init__(self):
+            self.w = TestWorker.remote()
+            with InputNode() as inp:
+                x = self.w.add_one.bind(inp)
+                y = self.w.add_one.bind(x)
```

Comment:
This is similar to `test_driver_and_intraprocess_read`. Can you add a test that is similar to `test_driver_and_actor_as_readers`, i.e., having another actor as reader of `x`

Reply:
added to the other new pr